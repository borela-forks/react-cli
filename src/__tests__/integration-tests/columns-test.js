// @flow strict

import React from "react";
import TestRender from "./test-render";
import { Section } from "../../index";

TestRender(
  "Horizontal sections should render as columns",
  <Section horizontal>
    1st column
    <br />
    2nd column
    <br />
    3rd column
  </Section>
);
