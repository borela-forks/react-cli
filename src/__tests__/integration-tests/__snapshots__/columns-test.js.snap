// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Horizontal sections should render as columns 1`] = `
"1st column++++++2nd column++++++3rd column++++++++
"
`;
